src/utils/lnurl.ts

```diff
@@ -312,14 +322,16 @@ export const handleLnurlWithdraw = async ({
 		return err(callbackRes.error.message);
 	}
 
-	const channelStatusRes = await fetch(callbackRes.value);
-	if (channelStatusRes.status !== 200) {
+	let channelStatusRes: any;
```

Comment:
To avoid `let channelStatusRes: any;`

```suggestion
try {
		const channelStatusRes = await fetch(callbackRes.value);
		const jsonRes = await channelStatusRes.json();

		if (jsonRes.status === 'ERROR') {
			showToast({
				type: 'error',
				title: i18n.t('other:lnurl_withdr_error'),
				description: jsonRes.reason,
			});
			return err(jsonRes.reason);
		}

		showToast({
			type: 'success',
			title: i18n.t('other:lnurl_withdr_success_title'),
			description: i18n.t('other:lnurl_withdr_success_msg'),
		});

		return ok({ type: EQRDataType.lnurlWithdraw });
	} catch (e) {
		showToast({
			type: 'error',
			title: i18n.t('other:lnurl_withdr_error'),
			description: e.message,
		});
		return err(e.message);
	}
```